build/package/Installer.RPM.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <rpmConfigJsonName>rpm_config.json</rpmConfigJsonName>
```

Comment:
The file you added below is called dotnet_config.json? Does it get transformed into rpm_config.json?